src/WarcraftLegacies.Source/Quests/Scourge/QuestSapphiron.cs

```diff
@@ -33,7 +34,7 @@ public sealed class QuestSapphiron : QuestData
       "Sapphiron has been slain, and has been reanimated as a mighty Frost Wyrm under the command of the Scourge.";
 
     /// <inheritdoc/>
-    protected override string RewardDescription => "Unlock one of the requirements to train Frost Wyrms. " +
+    protected override string RewardDescription => "Unlock ability to train Frost Wyrms. " +
```

Comment:
While you're in there can you change this to the standard format, i.e. "Learn to train x from the y", and use GetObjectName for x and y?

Reply:
Updated